consensus/progpow/consensus.go

```diff
@@ -383,6 +383,31 @@ func (progpow *Progpow) IsDomCoincident(chain consensus.ChainHeaderReader, heade
 	return order < common.NodeLocation.Context()
 }
 
+func (progpow *Progpow) DeriveMixAndPowHash(header *types.Header) (mixHash, powHash common.Hash) {
```

Comment:
This implementation is specifically the 'light' mode of ProgPoW. I think the name should include that. Maybe `ComputeLightPoW`